src/matrix/verification/SAS/channel/types.ts

```diff
@@ -0,0 +1,25 @@
+export const enum VerificationEventTypes {
```

Comment:
If I'm not mistaken, we usually use singular for enums?
```suggestion
export const enum VerificationEventType {
```